cypress/integration/dresses-list.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { MenuContentPage, DressesListPage } from "../page/index"
+
+
+describe("the user navigates to the dresses page should", () => {
+
+  let menuContentPage: MenuContentPage;
+  let dressesListPage: DressesListPage;
+
+  before(() => {
+    menuContentPage = new MenuContentPage();
+    dressesListPage = new DressesListPage();
+  })
+
+  it("show the available dresses", () => {
+    menuContentPage.visitMenuContentPage()
+    menuContentPage.goToDressesMenu()
+    dressesListPage.getDressProducts()
+    dressesListPage.validateItemsNumber(5)
+    dressesListPage.validateItemsNames(["Printed Dress", "Printed Dress", "Printed Summer Dress", "Printed Summer Dress", "Printed Chiffon Dress"])
```

Comment:
And then pass the array created in the arrange section here in the assert section.